src/lua/help_en_US.lua

```diff
@@ -7,8 +7,8 @@ To start the interactive Tarantool tutorial, type 'tutorial()' here.
 
 Available backslash commands:
 
-  \set language <language> -- for setting language (Lua by default)
-  \set delimiter <delimiter> -- for setting delimiter
+  \set language <language> -- for setting language (lua or sql)
+  \set delimiter <delimiter> -- for setting end-of-line delimiter
```

Comment:
It is a delimiter between a lua/sql statements. It is not correct to name it 'end-of-line delimiter'.